wrappers/flutter/lib/src/infra/fitatu_barcode_scanner_preview.dart

```diff
@@ -32,32 +41,52 @@ class FitatuBarcodeScannerPreview extends StatefulWidget {
 class FitatuBarcodeScannerPreviewState extends State<FitatuBarcodeScannerPreview> with ScannerPreviewMixin {
   late final _key = GlobalKey<ScannerPreviewMixin>();
 
+  @override
+  void initState() {
+    unawaited(SystemChrome.setPreferredOrientations([DeviceOrientation.portraitUp]));
+    super.initState();
+  }
+
+  @override
+  void dispose() {
+    unawaited(SystemChrome.setPreferredOrientations(widget.theme.supportedOrientations));
```

Comment:
a co się stanie jeśli ktoś zminimalizuje skanner i otworzy apkę z widgeta (wtedy czasami leci re-create activity chyba bo apka wstaje od nowa). To przejdzie przez dispose?

Reply:
Tak

Reply:
ok